cpp/include/cuml/cluster/hdbscan.hpp

```diff
@@ -485,4 +471,22 @@ void out_of_sample_predict(const raft::handle_t& handle,
                            int min_samples,
                            int* out_labels,
                            float* out_probabilities);
+
+void _compute_core_dists(const raft::handle_t& handle,
+                         const float* X,
+                         float* core_dists,
+                         size_t m,
+                         size_t n,
+                         raft::distance::DistanceType metric,
+                         int min_samples);
+
+void _compute_inverse_label_map(const raft::handle_t& handle,
```

Comment:
While I know these functions are only being used internally for cython at the moment, they are still being exposed through the C++ public hdbscan API. I'd prefer to remove the `_` prefix. Also, I don't see any particular reason they *shouldn't* be exposed publicly. One thing we can do if we really want to separate this from the training/inference public APIs is nest it inside a `helper` namespace like `::hdbscan::helper`